packages/components/scss/BaseStyleSheet.scss

```diff
@@ -764,7 +766,7 @@ input[type='number']::-webkit-inner-spin-button {
     background-image: escape-svg($custom-select-indicator),
       linear-gradient(0deg, $gray-700, $gray-700);
     &:-moz-focusring {
-      color: rgba(0, 0, 0, 0%);
+      color: transparent;
       text-shadow: 0 0 0 $foreground !important;
     }
   }
```

Comment:
`.console-creator` isn't needed on Community, console-creator is only on Enterprise (and we may not even need this clause at all anymore)

Reply:
I created https://github.com/deephaven/web-client-ui/issues/1635 to follow up on this one